kytos/core/tcp_server.py

```diff
@@ -95,6 +99,9 @@ def setup(self):
 
         self.connection = Connection(self.addr, self.port, self.request)
         server_port = self.server.server_address[1]
+        if self.server.protocol_name:
+            self.known_ports.update({server_port: self.server.protocol_name})
```

Comment:
Maybe I'm being too pedantic here but, in this case, the line below is more common and readable while having the exactly same behavior as the line above.
```python
self.known_ports[server_port] = self.server.protocol_name
```

Reply:
Definitely!